### ISSUE TITLE
FR: load tab on demand

### ISSUE THREAD

**Issue by wladmis** [philc/vimium#2902]
When browser is opened with a lot of tabs that are unloaded, navigating with J or K keys causes tab loading that can hang the navigation. It would be convenient to have an option to navigate tab focus without actual tab loading and when the tight tab is reached, load it on demand (for example with r key).

**Comment by GKFX** [comment]
It would probably be easier just to use `Vomnibar.activateTabSelection`, which is on <kbd>T</kbd> by default. That allows you to get to your desired tab in only a few keypresses (T, first few letters of tab name, return) rather than having to press J/K repeatedly.

**Comment by wladmis** [comment]
> It would probably be easier just to use Vomnibar.activateTabSelection, which is on T by default. That allows you to get to your desired tab in only a few keypresses (T, first few letters of tab name, return) rather than having to press J/K repeatedly.

You are right, it is convenient for a long-distance tabs, but for near-located tab it was easier to press J/K several times. This worked perfect with similar Fx addon (which is obsolete now) based on XUL, but it seems that it is easier to use Vomnibar.activateTabSelection instead indeed.

**Comment by smblott-github** [comment]
What does "tabs unloaded" mean?  If it means that you're using [the great suspender](https://chrome.google.com/webstore/detail/the-great-suspender/klbibkeccnjlkjkiokjodocebajanakg?hl=en), then there is nothing we can do.  Chrome (FF) does not run extensions like Vimium on other extensions' pages.

**Comment by wladmis** [comment]
> What does "tabs unloaded" mean?
The tabs that are not loaded yet. For example, when you start browser and restore previous session which contains tabs, all of them except current one are not loaded. And when you move tab focus on another tab, the tab is loading. This is current behavior.
I propose a feature that tab focus moving just move the focus without actual newly focused tab loading.
